f2/src/forms/__tests__/HowDidItStartForm.test.js

```diff
@@ -71,6 +71,6 @@ describe('<HowDidItStartForm />', () => {
       await wait(0) // Wait for promises to resolve
     })
 
-    expect(submitMock).toHaveBeenCalledTimes(0)
+    expect(submitMock).toHaveBeenCalledTimes(1)
```

Comment:
The name of this test "does not call the onSubmit function when the form is submitted" should be changed to "call the onSubmit function when the form is submitted"